packages/server/lib/browsers/firefox.ts

```diff
@@ -510,7 +510,14 @@ export async function open (browser: Browser, url, options: any = {}, automation
   })
 
   try {
-    await firefoxUtil.setup({ automation, extensions: launchOptions.extensions, url, foxdriverPort, marionettePort, remotePort, onError: options.onError })
+    await firefoxUtil.setup({ automation,
+      extensions: launchOptions.extensions,
```

Comment:
i have never seen this coding style before :smile: 

```suggestion
    await firefoxUtil.setup({
      automation,
      extensions: launchOptions.extensions,
```

Reply:
It's a pretty advanced style, but maybe the world isn't ready for it yet. Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/0569c91b37e802fc2e7613649c8813c03dbbc8c0